platforms/hyperledger-fabric/configuration/roles/helm_component/templates/invoke_chaincode_job.tpl

```diff
@@ -14,8 +14,6 @@ spec:
   values:
     metadata:
       namespace: {{ namespace }}
-      network:
```

Comment:
As both the fabric versions, the 1.4.x and 2.x shares the same invoke chart, so the network version is required.